src/phoenix/server/main.py

```diff
@@ -160,7 +160,7 @@ def _load_items(
             args=(traces, fixture_spans, simulate_streaming),
             daemon=True,
         ).start()
-        fixture_evals = get_evals_from_fixture(trace_dataset_name)
+        fixture_evals = list(get_evals_from_fixture(trace_dataset_name))
```

Comment:
For my understanding, this just forces the generator to materialize the data?

Reply:
yes, the the network call happens in the main thread